package.json

```diff
@@ -24,11 +25,13 @@
     "@nimiq/style": "^0.8.2",
     "@nimiq/utils": "^0.8.2",
     "big-integer": "^1.6.44",
+    "ethers": "^5.7.2",
```

Comment:
That's a huge dependency for just checking address validity. There's smaller packages for that, such as
- https://www.npmjs.com/package/ethereum-address (really only that one function)
- https://www.npmjs.com/package/@ethersproject/address (part of ethers, with more stuff)

I would much prefer to use such a small package and include it in the bundle, and not require the using-project to install and bundle ethers downstream.

Reply:
I guess it makes sense to use something smaller than `ethers` and only for address validation, but since the package is already loaded anyway in the wallet we though it would be alright to use it there. But now that I think about it, if someone else use the component outside of the wallet and use another library than `ethers` then it would become quite a heavy component just for address validation.
So I will change the component to actually include the function `isAddress` and `isChecksumAddress` since they are quite short, and now only use the `crypto-js` package who's still needed for the `sha3` method used in the `isChecksumAddress` function.